fanaSystem/fanaSystem/settings.py

```diff
@@ -69,6 +97,8 @@
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
     "corsheaders.middleware.CorsMiddleware",
+    "fanaSystem.middlewares.RouteSpecificMiddleware",
+    "fanaSystem.middlewares.ErrorHandlingMiddleware",
```

Comment:
What's the usage of ErrorHandlingMiddleware?
Can you document it in the comments to track it later?

Reply:
This will just make debugging easier and organised , and we may later add pages for specific errors.